components/brave_news/browser/resources/feed/Card.tsx

```diff
@@ -103,14 +103,15 @@ interface BraveNewsLinkProps extends SecureLinkProps {
   feedDepth?: number
 }
 
-export function BraveNewsLink(props: BraveNewsLinkProps) {
+export function BraveNewsLink({ feedDepth, ...rest }: BraveNewsLinkProps) {
+  const props = { feeddepth: feedDepth, ...rest }
```

Comment:
```
Warning: React does not recognize the `feedDepth` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `feeddepth` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

Aha, it's common component so we can't declare the react prop for it, so change casing. I'm curious what if we don't pass props to the `SecureLink` or just passing what it needs like href or intersection between AnchorHTMLAttributes and `BraveNewLInkProps`

Reply:
The p3a team added this - we actually want the prop to go through to the HTML element and set an attribute, because the attribute gets read on clicks

Reply:
Aha makes sense. Thanks for the explanation.